packages/itensor/quri_parts/itensor/estimator.py

```diff
@@ -56,17 +58,25 @@ def _estimate(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
     op = convert_operator(operator, s)
 
     # calculate expectation value
-    psi = jl.apply(circuit, psi)
+    psi = jl.apply(circuit, psi, **kwargs)
     exp: float = jl.expectation(psi, op)
 
     return _Estimate(value=exp, error=0.0)
 
 
-def create_itensor_mps_estimator() -> QuantumEstimator[ITensorStateT]:
+def create_itensor_mps_estimator(**kwargs: Any) -> QuantumEstimator[ITensorStateT]:
     """Returns a :class:`~QuantumEstimator` that uses ITensor MPS simulator to
-    calculate expectation values."""
+    calculate expectation values.
+
+    Keyword arguments are passed to `ITensors.apply
+
+    <https://itensor.github.io/ITensors.jl/dev/MPSandMPO.html#ITensors.product-Tuple{ITensor,%20ITensors.AbstractMPS}>`_
+    """
+
+    def estimator(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
+        return _estimate(operator, state, **kwargs)
```

Comment:
There might be a better implementation though
```suggestion
    Args:
        mindim: The minimum number of singular values. The value is passed to `ITensors.apply`.
        maxdim: ...
        cutoff: ...
    Keyword arguments are passed to `ITensors.apply

    <https://itensor.github.io/ITensors.jl/dev/MPSandMPO.html#ITensors.product-Tuple{ITensor,%20ITensors.AbstractMPS}>`_
    """

    def estimator(operator: Estimatable, state: ITensorStateT) -> Estimate[complex]:
        if mindim:
            kwargs["mindim"] = mindim
        if ...
        
        return _estimate(operator, state, **kwargs)
```

Reply:
Thank you. This is better than mine.